serverless/pytorch/foolwood/siammask/nuclio/function.yaml

```diff
@@ -18,10 +18,20 @@ spec:
 
   build:
     image: cvat/pth.foolwood.siammask
-    baseImage: continuumio/miniconda3
+    baseImage: ubuntu:20.04
```

Comment:
Changed base image to have the same set of preCopy directives.
But finally it was nesessary to add
``pip install torch==1.9.0+cu111 torchvision==0.10.0+cu111 torchaudio==0.9.0 -f https://download.pytorch.org/whl/torch_stable.html``
to function-gpu.yml, so it is not the same now anyway. Let me know if you think that better to leave this file as is.